designs/developer/add-45/sections/Adv-Health_Checks.md

```diff
@@ -5,4 +5,63 @@ version: 4.5
 
 # Add Flows - Advanced Options - Health Checks
 
-coming soon
+Health Checks is a link within the Advanced Options section for the Import from Git and Deploy Image +Add flows. The Health Checks link goes after the Routing link and before the Build Configuration link in the list of Advanced Option links.
+
+![Health Checks link](img/Select_Health_Checks.png)
+
+
+When a user clicks on the Health Checks link, the Health Checks section is expanded within the Advanced Options section.
+
+![Health Checks link clicked](img/Select_Health_Checks_2.png)
+
+When the Health Checks section is expanded there are 3 probes listed (Readiness, Liveness, and Startup) with a Add <probe name> link associated with each probe.
+
+![Health Checks options](img/Select_Add_Probe.png)
+
+When an  Add <probe name>  link is clicked, the probe form expands exposing all of the fields associated with the probe.  The user can confirm or cancel the adding of the Health Check using the check or x.  There are 3 variations on the Health Checks form which are based on the Type that is selected. The default probe type is HTTP GET. See the Health Checks In-Line form section below for these variations.
+
+![Health Checks inline form](img/Add_Probe_Inline_Form_Expands.png)
+
+After the in-line form has been validated and the probe has been added, the form contracts and displays <probe name> Probe Added”. The added probe can be removed after it has been added by clicking on the minus sign. An Edit Probe link can be used to review the entered values.
```

Comment:
```suggestion
After the in-line form has been validated and the probe has been added, the form contracts and displays <probe name> Probe”. The added probe can be removed after it has been added by clicking on the minus sign. An Edit Probe link can be used to review the entered values.
```

Reply:
Suggesting a change here to align with the design...or maybe the design needs to be updated to include "Added"?

Reply:
@lizsurette thanks! fixed description.